src/plugin/src/main/java/com/defenseunicorns/uds/keycloak/plugin/clientpolicy/executor/UDSClientPolicyPermissionsExecutor.java

```diff
@@ -55,6 +61,8 @@ public class UDSClientPolicyPermissionsExecutor implements ClientPolicyExecutorP
             UserAttributeStatementMapper.PROVIDER_ID,
             UserAttributeMapper.PROVIDER_ID,
             UserPropertyAttributeStatementMapper.PROVIDER_ID,
+            GroupMembershipMapper.PROVIDER_ID,
+            AudienceProtocolMapper.PROVIDER_ID, // tbd: should we add this by default to support https://github.com/uds-packages/argo-workflows/blob/5257c03814a9f6b3925e26aaaea914918344dc37/chart/templates/uds-package.yaml#L26
```

Comment:
I think I'm leaning towards adding this to support the current use-cases (this was found by searching for protocolMappers across our orgs). We could revisit this in the future potentially, but I'd rather make things work out of the box here for our users.
```suggestion
            AudienceProtocolMapper.PROVIDER_ID,
```

Curious on @slaskawi and @UnicornChance thoughts...

Reply:
I don't see why not. We're not supplying the configuration out of the box, so it will take a few extra hoops for someone to make use of this "allowed" aud client protocol. Two areas of concern are misconfiguration and technically its not least privileged configuration, but i really don't think either of those are good enough reasons to not add it.

Reply:
Yep, let's add it!